content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -276,6 +276,33 @@ not affect the performance of the member.
 
 Below is an example for taking a snapshot of the keyspace served by
 `$ENDPOINT` to the file `snapshot.db`:
+step1:Take an etcd snapshot backup using the following command
+
+ETCDCTL_API=3 etcdctl \
+  --endpoints=https://127.0.0.1:2379 \
+  --cacert=<ca-file> \
+  --cert=<cert-file> \
+  --key=<key-file> \
+  snapshot save <backup-file-location>
+
+  Step2 : How do we get the endpoint and the certificate informatio
+  we can retrieve them from the etcd pod and the manifest file for etcd pod is located under /etc/kubernetes/manifests folder
```

Comment:
I'd use an [ordered list](https://www.google.com/search?q=ordered+list+in+Markdown) item here.